app/assets/stylesheets/_manuals-print.scss

```diff
@@ -60,7 +60,11 @@ a {
   margin-bottom: 30px;
 
   h2 {
-    @include core-16;
```

Comment:
Sorry, confused. Why is this not being replaced with a `govuk-font` mixin?

Reply:
this stylesheet isn't importing govuk-frontend and the font still comes from govuk_template
will have a check again an do a selective import from govuk-frontend